<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>272</width>
    <height>480</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>OpenIVI Mobility HTML5</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="WebGraphicView" name="graphicsView">
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>200</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::AdjustToContents</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar_">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>272</width>
     <height>30</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionOpen_URL"/>
    <addaction name="actionBluetooth"/>
    <addaction name="separator"/>
    <addaction name="action_Quit"/>
   </widget>
   <widget class="QMenu" name="menu_Window">
    <property name="title">
     <string>&amp;Window</string>
    </property>
    <addaction name="action_Full_Screen"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Window"/>
  </widget>
  <widget class="QStatusBar" name="statusBar_"/>
  <action name="action_Quit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
  </action>
  <action name="actionOpen_URL">
   <property name="text">
    <string>&amp;Open File...</string>
   </property>
  </action>
  <action name="action_Full_Screen">
   <property name="text">
    <string>&amp;Full Screen (F11)</string>
   </property>
   <property name="iconText">
    <string>Full Screen</string>
   </property>
  </action>
  <action name="action_No_Data">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Stop</string>
   </property>
   <property name="iconText">
    <string>Stop</string>
   </property>
  </action>
  <action name="actionFrom_file">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>From file</string>
   </property>
  </action>
  <action name="actionFrom_CAN_bus">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>From CAN bus</string>
   </property>
   <property name="toolTip">
    <string>From CAN bus</string>
   </property>
  </action>
  <action name="actionBluetooth">
   <property name="text">
    <string>Manage Bluetooth Devices...</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>WebGraphicView</class>
   <extends>QGraphicsView</extends>
   <header>webgraphicview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
